packages/node_modules/pouchdb-core/src/setup.js

```diff
@@ -127,4 +128,54 @@ PouchDB.fetch = function (url, opts) {
   return fetch(url, opts);
 };
 
+function ActiveTasks() {
+  this.tasks = [];
```

Comment:
This could also be an `{}` so all `this.tasks[id]` operations become O(1) instead of O(n). Are there downsides?

Reply:
Ooh, good idea. Not off the top of my head. @jankaszel ?

Reply:
i agree with @janl, we're not relying on the order of the `tasks` variable. all methods (`get`, `remove`, `update`) work with an ID, so we could indeed get O(1) access here instead of using `tasks.findIndex()`!